frontend/src/components/molecules/AddTransaction.tsx

```diff
@@ -22,39 +74,50 @@ const AddTransaction: React.FC<{ className?: string }> = props => {
     interval_type,
     interval,
   }: any) => {
-    if (!recurring) {
-      await TransactionActions.doCreateTransaction(
-        {
-          company_id: auth!.selectedCompany!,
-          date,
-          description,
-          money: money * 100,
-          notes,
-          type,
-        },
-        dispatch
-      );
-    } else {
-      await TransactionActions.doCreateRecurringTransaction(
-        {
-          company_id: auth!.selectedCompany!,
-          end_date,
-          interval,
-          interval_type,
-          start_date: date,
-          template: {
+    if (!(description.length > 140) && !(notes.length > 140)) {
```

Comment:
Here it's probably better to use guard statements. So instead of wrapping everything in an `if`-statement, just check your stuff, do your "YOU DID NOT ENTER THINGS ACCORDING TO HOW I WANT THEM"-snack-dispatching, and then `return;`. That way, you don't have to indent everything and increase the "legibility"-complexity.

```typescript
if (!validLength) {
  toast('invalid length');
  return;
}

if (!validFormat) {
  toast('invalid format');
  return;
}

// etc.
```